android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/util/DemoUtil.java

```diff
@@ -43,4 +46,14 @@ public static void setLightStatusBar(Context context, Window window) {
 		window.setStatusBarColor(ContextCompat.getColor(context, R.color.login_status_bar_color));
 		window.getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR);
 	}
+
+	public static void showNegativeSnackBar(Context context, View layout) {
```

Comment:
@camisLuisa I believe the name should be `showErrorSnackbar` but you can change this in other PR

Reply:
okay